kotest-extensions/kotest-extensions-quarkus/src/jvmTest/kotlin/com/sksamuel/kt/quarkus/QuarkusListenerTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.sksamuel.kt.quarkus
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.quarkus.test.junit.QuarkusTest
+import javax.inject.Inject
+
+@QuarkusTest
+class QuarkusListenerTest : WordSpec() {
+   @Inject
+   var service: UserService? = null
+
+   init {
+      "QuarkusListener" should {
+         "have injected the service" {
+            service.shouldNotBeNull()
+         }
+      }
+   }
+}
```

Comment:
What would be the syntax for inline spec?
For example
```
@QuarkusTest
class QuarkusListenerTest : WorSpec({
    @Inject
    var service: UserSerive? = null

    "QuarkusListener" should {
        "have injected the service" {
            service.shouldNotBeNull()
        }
    }

})
```

Would this already work?